### PR TITLE
footer done for mobile and desktop

### PR DESCRIPTION
**Pull request by IvailoStoyanov**

### CODE REVIEW

File: src/scaffolding/master.scaffolding.hbs [hunk 1]
```diff
@@ -13,4 +13,21 @@
...
+        <div class="swooshFoldWrapper__swooshFold"></div>
+    </div>
+        <img class="footer__githubLogo" src="_assets/github-logo.svg" alt="github octo-cat">
+<div class="footer__checkItOut">
+    <p>Of course we also built<br><span class="footer__bigWords">this website</span> with biotope.</p>
+            <!-- placeholder buttons - will be replaced with bio components -->
+        <bio-button title="Check it out" modifier="blue" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
+    </div>
+<div class="footer__essentials">
+    <a href="#">Impritn</a>
```
Comment (luke-m) [inline src/scaffolding/master.scaffolding.hbs]: Imprint*

File: src/scaffolding/master.scaffolding.hbs [hunk 2]
```diff
@@ -13,4 +13,21 @@
             <button class="splash__button splash__button--blue">See Docs</button>
         </div>
     </div>
-</header>
\ No newline at end of file
+</header>
+
+<footer class="footer">
+    <div class="swooshFoldWrapper">
```
Comment (luke-m) [inline src/scaffolding/master.scaffolding.hbs]: We need to talk about this implementation I think. I'm not sure if I approve of this but we can merge it for now.

File: src/resources/scss/global.scss [hunk 3]
```diff
@@ -55,4 +55,116 @@
...
     }
+}
+
+.footer{
+    position: relative;
+    padding: ($includingSwooshPadding + 50px) 20px 0 20px;
+    background: $bio-gray;
+    color: $black;
+
+    @media screen and (min-width: $medium) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($medium)` for this please.

File: src/resources/scss/global.scss [hunk 4]
```diff
@@ -55,4 +55,116 @@
...
+        display: flex;
+        justify-content: space-between;
+        padding: $includingSwooshPadding 10vw 0 10vw;
+    }
+    
+    @media screen and (min-width: $large) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($large)` for this please.

File: src/resources/scss/global.scss [hunk 5]
```diff
@@ -55,4 +55,116 @@
...
+        padding: $includingSwooshPadding 15vw 0 15vw;
+    }
+
+    @media screen and (min-width: $xlarge) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($xlarge)` for this please.

File: src/resources/scss/global.scss [hunk 6]
```diff
@@ -55,4 +55,116 @@
...
+    
+    
+    &__githubLogo{
+        position: absolute;
+        top: 30px;
+        left: 0;
+        transform: rotate(90deg);
+        width: 70px;
+
+        @media screen and (min-width: $large) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($large)` for this please.

File: src/resources/scss/global.scss [hunk 7]
```diff
@@ -55,4 +55,116 @@
...
+            left: auto;
+            bottom: 0;
+            transform: rotate(0);
+            width: auto;
+        }
+    }
+
+    &__checkItOut{
+        padding-top: 25px;
+        line-height: 1.3;
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Consider exporting this typeface to `typography.scss` please.

File: src/resources/scss/global.scss [hunk 8]
```diff
@@ -55,4 +55,116 @@
...
+
+        @media screen and (min-width: $medium) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($medium)` for this please.

File: src/resources/scss/global.scss [hunk 9]
```diff
@@ -55,4 +55,116 @@
...
+            flex-grow: 1;
+            padding: 0 3vw;
+            font-size: 20px;
+        }
+        @media screen and (min-width: $large) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($large)` for this please.

File: src/resources/scss/global.scss [hunk 10]
```diff
@@ -55,4 +55,116 @@
...
+            padding-left: 17vw;
+        }
+    }
+    
+    &__bigWords{
+        font-family: $font-family-header;
+        font-size: 30px;
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Consider exporting this typeface to `typography.scss` please.

File: src/resources/scss/global.scss [hunk 11]
```diff
@@ -55,4 +55,116 @@
...
+        text-transform: uppercase;
+         @media screen and (min-width: $medium) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($medium)` for this please.

File: src/resources/scss/global.scss [hunk 12]
```diff
@@ -55,4 +55,116 @@
...
+    
+    svg{
+        background: beige;
+    }
+
+    &__essentials{
+        display: flex;
+        flex-flow: column;
+        padding: 40px 0;
+        font-weight: $font-weight-bold;
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Consider exporting this typeface to `typography.scss` please.

File: src/resources/scss/global.scss [hunk 13]
```diff
@@ -55,4 +55,116 @@
...
+
+        a {
+            margin-bottom: 10px;
+            color: $black;
+            text-decoration: none;
+            &:hover{
+                text-decoration: underline;
+            }
+
+            @media screen and (min-width: $large) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: Use `@include breakpoint($large)` for this please.

File: src/scaffolding/master.scaffolding.hbs (hunk 2 above)
Comment (IvailoStoyanov) [reply src/scaffolding/master.scaffolding.hbs]: Let's talk 👍

File: src/resources/scss/global.scss (hunk 10 above)
Comment (IvailoStoyanov) [reply src/resources/scss/global.scss]: Would this mean we move all the fonts from _settings.scss to typography.scss?

Comment (luke-m) [reply src/resources/scss/global.scss]: No, that's just the place for extendable typefaces such as %benefit-text.

File: src/resources/scss/global.scss [hunk 14]
```diff
@@ -55,4 +56,101 @@
...
+        display: flex;
+        justify-content: space-between;
+        padding: $includingSwooshPadding 10vw 0 10vw;
+    }
+    
+    @include breakpoint ($large) {
+        padding: $includingSwooshPadding 15vw 0 15vw;
+    }
+
+    @include breakpoint ($xlarge) {
```
Comment (luke-m) [inline src/resources/scss/global.scss]: no space between `breakpoint` and the parenthesis please

File: src/resources/scss/global.scss [hunk 15]
```diff
@@ -55,4 +56,101 @@
...
+            flex-grow: 1;
+            padding: 0 3vw;
+            font-size: 20px;
+        }
+        @include breakpoint ($large) {
+            padding-left: 17vw;
+        }
+    }
+
+    &__essentials{
```
Comment (luke-m) [inline src/resources/scss/global.scss]: space between classname and curly braces please

**Pull request merged**